src/main/java/se/knowit/bookitnotification/servicediscovery/AwsDiscoveryServiceImpl.java

```diff
@@ -13,7 +13,7 @@ public AwsDiscoveryServiceImpl(AWSServiceDiscovery serviceDiscoveryClient) {
     }
 
     @Override
-    public DiscoveryServiceResult discoverInstances(String serviceName) {
+    public DiscoveryServiceResult discoverInstances(String namespaceName, String serviceName) {
         DiscoveryServiceResult result = new DiscoveryServiceResult();
         DiscoverInstancesResult discoverInstancesResult = serviceDiscoverClient.discoverInstances(
                 new DiscoverInstancesRequest().withServiceName(serviceName));
```

Comment:
shouldn't namespaceName be part of the request? e.g new DiscoverInstancesRequest().withNamespace(namespaceName).withServiceName(serviceName)

Reply:
Yup it should. Hmm..where did that go...